ansible/roles/common/tasks/main.yml

```diff
@@ -63,3 +73,14 @@
     dest: /etc/nginx/nginx.conf
     regexp: 'server_names_hash_bucket_size'
     line: '	server_names_hash_bucket_size 64;'
+
+- name: nginx default site conf
+  template: src=nginx/default
+            dest=/etc/nginx/sites-enabled/default
+
+- name: remove default nginx site html page
```

Comment:
Right, but:
* you're already replacing the default nginx site
* when ansible is run the 2nd time, it will remove `index.html` and re-add it, which is a waste